src/main/resources/doc/en/html/libs/gates/controlled.html

```diff
@@ -57,10 +57,10 @@ <h1>
                 <tbody>
 				  <tr>
 				     <td>
-                      <img class="appearancelibs" src="../../../../img-libs/anscontrolledBuffer.png" alt="#########" height="64" width="64">
+                      <img class="appearancelibs" src="../../../../img-libs/anscontrolledbuffer.png" alt="#########" height="64" width="64">

```

Comment:
Please remove `alt="#########"`(multiple places)

Reply:
... or replace by a descriptive text als alternative representation of the image.